patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -963,7 +963,7 @@
  				KillTile(num + 1, num6 + m);
  		}
  
-+		if (!TileLoader.Drop(num, j, type))
++		if (!TileLoader.Drop(i, j, type))
```

Comment:
Could we consider a `using var` hack which disables all drops until the end of the current scope? See usages of `Item.DisableCloneMethod` and `Main.CurrentPlayerOverride` for examples. Might reduce patch size and improve maintainability dramatically.